unit/src/TabulatedFluidPropertiesTest.C

```diff
@@ -120,3 +122,25 @@ TEST_F(TabulatedFluidPropertiesTest, fromFile)
   REL_TEST("cv", _tab_fp->cv(p, T), _co2_fp->cv(p, T), 1.0e-4);
   REL_TEST("entropy", _tab_fp->s(p, T), _co2_fp->s(p, T), 1.0e-4);
 }
+
+// Test generation of tabulated fluid properties
+TEST_F(TabulatedFluidPropertiesTest, generateTabulatedData)
+{
+  // We always want to generate a new file in this test, delete existing file
+  std::remove("fluid_properties.csv");
+
+  Real p = 1.5e6;
+  Real T = 450.0;
+
+  // Generate the tabulated data
+  const_cast<TabulatedFluidProperties *>(_tab_gen_fp)->initialSetup();
```

Comment:
Could this be moved into `SetUp()`? It is part of the setup rather then the test.

Reply:
I tried this, but it seems that `SetUp()` is called before each test in this file, so this is setup multiple times.

There is a `SetUpTestCase()` that from reading the documentation is called only once, but I couldn't get it to work properly.

Reply:
That sucks :-( Thanks for trying...